[Wt] build fail

C:\_\3rd\vcpkg>vcpkg version
Vcpkg package management program version 0.0.61-ed2994a5e930b27fd5dbe121b35fe0abba428376

See LICENSE.txt for license information.


output message: 
-- CURRENT_INSTALLED_DIR=C:/_/3rd/vcpkg/installed/x64-windows
-- DOWNLOADS=C:/_/3rd/vcpkg/downloads
-- CURRENT_PACKAGES_DIR=C:/_/3rd/vcpkg/packages/wt_x64-windows
-- CURRENT_BUILDTREES_DIR=C:/_/3rd/vcpkg/buildtrees/wt
-- CURRENT_PORT_DIR=C:/_/3rd/vcpkg/ports/wt/.
-- Using cached C:/_/3rd/vcpkg/downloads/wt-3.3.6.tar.gz
-- Testing integrity of cached file...
-- Testing integrity of cached file... OK
-- Extracting done
-- Configuring x64-windows-rel
-- Configuring x64-windows-rel done
-- Configuring x64-windows-dbg
-- Configuring x64-windows-dbg done
-- Package x64-windows-rel
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:14 (message):
    Command failed: C:/Program Files/CMake/bin/cmake.exe;--build;.;--config;Release;--target;install;--;/m
    Working Directory: C:/_/3rd/vcpkg/buildtrees/wt/x64-windows-rel
    See logs for more information:
      C:\_\3rd\vcpkg\buildtrees\wt\package-x64-windows-rel-out.log
      C:\_\3rd\vcpkg\buildtrees\wt\package-x64-windows-rel-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_install_cmake.cmake:3 (vcpkg_execute_required_process)
  ports/wt/portfile.cmake:24 (vcpkg_install_cmake)
  scripts/ports.cmake:74 (include)


error output (in rel-out.log): 
LINK : fatal error LNK1104: cannot open file“boost_date_time.lib” [C:\_\3rd\vcpkg\buildtrees\wt\x64-windows-rel\src\Wt\Dbo\wtdbo.vcxproj]

--- a/ports/boost/portfile.cmake
+++ b/ports/boost/portfile.cmake
@@ -100,7 +100,6 @@ if (VCPKG_LIBRARY_LINKAGE STREQUAL dynamic)
         "\n#define BOOST_ALL_DYN_LINK\n"
     )
 endif()
-file(APPEND ${CURRENT_PACKAGES_DIR}/include/boost/config/user.hpp "\n#define BOOST_AUTO_LINK_NOMANGLE\n")
 
 file(INSTALL ${SOURCE_PATH}/LICENSE_1_0.txt DESTINATION ${CURRENT_PACKAGES_DIR}/share/boost RENAME copyright)
 message(STATUS "Packaging headers done")

